src/app/lib/utilities/getToggles/index.js

```diff
@@ -12,6 +12,27 @@ import getOriginContext from '#contexts/RequestContext/getOriginContext';
 const logger = nodeLogger(__filename);
 const NS_PER_SEC = 1e9;
 
+const logResponseTime = async (url, origin, service, timeout) => {
+  const isBrowser =
+    typeof window !== 'undefined' && typeof window.document !== 'undefined';
```

Comment:
Is there another way to implement `isBrowser` without passing in 'undefined' as a value? I can see we are using `undefined` as a value elsewhere in Simorgh to get contexts, it just feels strange.

Reply:
Tried different approaches but doesn't look like there is as `window` isn't defined.